staff/RominaFior/cv/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+<body>
+    <H1>Curriculum Vitae</H1>
+    <h2>Jesica Romina Sanchez</h2>
+    <p>Co Founder at Eqosis</p>
+    <p>Madrid y alrededores, EspañaServicios financieros</p>
+    
+    <h3>Extracto</h3>
+    <p>Internationally focused, strategic and results oriented professional, possessing more than a decade of professional experience in Australia, UK, and in Europe. Primarily experienced in financial, operational and risk analysis spanning across a broad range of geographic and demographic client profiles in both developed and emerging markets. Proven track record in project management through utilising analytical, leadership, communication and time management skills in order to develop and implement business strategy and position organisations for long term success.</p>
```

Comment:
Everything's in Spanish besides this summary. FIX IT 🗡

Reply:
👍 👌